.env.example

```
@@ -1,17 +1,20 @@
-# API Configuration
+# Environment Configuration Example
+# Copy this file to .env and fill in the values before running the application
+
+# Node Environment
+NODE_ENV=# local, development, production
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)